packages/wpcom-template-parts/src/universal-footer-navigation/index.tsx

```diff
@@ -121,14 +121,6 @@ export const PureUniversalNavbarFooter = ( {
 										{ __( 'P2: WordPress for Teams', __i18n_text_domain__ ) }
 									</a>
 								</li>
-								<li>
-									<a href="https://wpvip.com/" data-is_external="1" target="_self">
-										{ __( 'Enterprise', __i18n_text_domain__ ) }{ ' ' }
-										<span className="lp-link-chevron-external">
-											{ __( 'Solutions', __i18n_text_domain__ ) }
-										</span>
-									</a>
-								</li>
```

Comment:
I've renamed `Enterprise Solutions` to `Enterprise` to match the label used in the header. I've also move it below `Website Design Services` to be consistent with footers from landing pages (such as https://wordpress.com/support/).